pyt/analysis_base.py

```diff
@@ -38,7 +38,7 @@ def equal(self, value, other):
     def build_lattice(self, cfg):
         pass
 
+    @abstractmethod
```

Comment:
It's implemented in both classes that inherit from it, so might as well makes it abstract.